docs/extend/extend-apiml/api-medation-sso-integration-extenders.md

```diff
@@ -135,4 +137,35 @@ authentication:
 
 :::tip
 For more information about z/OSMF Single Sign-on, see [Establishing a single sign-on environment](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zosmfcore.multisysplex.help.doc/izuG00hpManageSecurityCredentials.html).
-:::
\ No newline at end of file
+:::
+
+## Forwarding x509 client certificate
+When client uses a x509 client certificates for authentication, the certificate can be forwarded to a downstream service. This is an alternative to the [Bypassing authentication for the service](#bypassing-authentication-for-the-service) option for client certificates.
+
+The following steps outline the x509 client certificate forwarding flow:
+
+1. A client sends a request to the API Gateway secured by a client certificate.
+2. The API Gateway extracts the client certificate from the request and puts this certificate into the `Client-Cert` header.
+3. The API Gateway forwards the request to the downstream service. The request is secured by the Zowe server certificate.
+4. The downstream service must validate the Zowe server certificate in the request to verify the certificate's origin and extract the original client certificate from the header. This validation makes it possible for the service to then perform the authentication.
+
+Both the API ML Gateway and the downstream service must conform to the following requirements to support x509 certificate forwarding:
+
+### API ML Gateway Requirements
+
+- Enable client cetificate forwarding in the `zowe.yaml`
```

Comment:
I think we should be explicit with our instructions.  For example, we state to enable client certificate forwarding in the `zowe.yaml` but don't state how to do that.  Instead, we show a yaml file and the reader must discern that the setting `components.gateway.apiml.security.x509.enabled` should be set to `true`.  

I like the idea of providing a reference / example of the yaml as shown, but I don't think references / examples should replace explicit instructions within the doc.